godotsteam.cpp

```diff
@@ -5175,15 +5066,14 @@ int Steam::configureConnectionLanes(uint32 connection, int lanes, Array prioriti
 Dictionary Steam::getCertificateRequest() {
 	Dictionary cert_information;
 	if (SteamNetworkingSockets() != NULL) {
-		int *certificate = new int[512];
-		int cert_size = 0;
+		int certificate[512];
+		int cert_size = sizeof(certificate);
 		SteamNetworkingErrMsg error_message;
-		if (SteamNetworkingSockets()->GetCertificateRequest(&cert_size, &certificate, error_message)) {
+		if (SteamNetworkingSockets()->GetCertificateRequest(&cert_size, certificate, error_message)) {
 			cert_information["certificate"] = certificate;
```

Comment:
Gives a warning of `warning: the address of 'certificate' will never be NULL [-Waddress]`.  Might be a GCC thing.

Reply:
Hmm. I get the same. I'm not sure what's going on with that warning since it doesn't give any hints about how it's being called. It should probably be a `PackedByteArray` anyway since it's a block of data and that's what `setCertificate()` uses, so I'll fix that up.

Reply:
That makes sense.  Cheers for the fixes!